angr/procedures/win32/GetLastInputInfo.py

```diff
@@ -4,19 +4,17 @@
 l = logging.getLogger(name=__name__)
 #l.setLevel("DEBUG")
 
-'''
-BOOL GetLastInputInfo(
-  PLASTINPUTINFO plii
-);
-
-typedef struct tagLASTINPUTINFO {
-  UINT  cbSize;  // The size of the structure, in bytes.
-                 // This member must be set to sizeof(LASTINPUTINFO).
-  DWORD dwTime;
-} LASTINPUTINFO, *PLASTINPUTINFO;
-
-'''
 class GetLastInputInfo(angr.SimProcedure):
+    """ BOOL GetLastInputInfo(
+      PLASTINPUTINFO plii
+    );
+
+    typedef struct tagLASTINPUTINFO {
+      UINT  cbSize;  // The size of the structure, in bytes.
+                     // This member must be set to sizeof(LASTINPUTINFO).
+      DWORD dwTime;
+    } LASTINPUTINFO, *PLASTINPUTINFO;
+    """
```

Comment:
This is not an appropriate use of a docstring, please just have it be a block comment